config-wdio/local.conf.ts

```diff
@@ -0,0 +1,37 @@
+var defaults = require("./base.conf").config;
+var _ = require("lodash");
+import * as configVal from "config";
+const BASEURL = configVal.get("Environment.baseUrl");
+const BROWSER = process.env.BROWSER||'chrome';
+let DRIVERS;
+let browserName;
+if (BROWSER.toLowerCase() === "chrome") {
+  (browserName = "chrome"), (DRIVERS = { chrome: { version: "87.0.4280.20" } });
```

Comment:
Instead of passing static version values, would you pass this value dynamically. If possible, can we keep it to latest?

Reply:
As of now it is not working with latest string, tried multiple options. For now, we are going ahead with static version. Will update in future if we find any better solution.